lint.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+cd frontend && npx tslint --project .
```

Comment:
I think the two commands in this script should be moved directly into .travis.yml. (The only reason test.sh exists is because it's more complex.) This also needs to `cd ..`